static/menu.js

```diff
@@ -235,4 +235,17 @@
         menu.firstElementChild.addEventListener("click", menuOnClick);
     }
     document.documentElement.addEventListener("keydown", menuKeyDown);
+    document.documentElement.addEventListener("keydown", function(ev) {
+        if (ev.key == "y") {
+            let permalink = document.getElementById("permalink");
+            if (document.location.hash != "") {
+              permalink.href += "#" + document.location.hash;
+            }
+            // Note: It would be nicer to do history.replaceState here and avoid a page load
+            // (that's what GitHub does), but the permalink goes through a redirect, so you wind up
+            // with a weird URL like:
+            // http://localhost:3000/crate/regex/1.3.1/target-redirect/x86_64-unknown-linux-gnu/regex/index.html
```

Comment:
Could we change it to not do this? The main reason for going through target-redirect in the first place was in case the page didn't exist on another platform or version, but we know that's never the case here.